docs/team/sp4ce-cowboy.md

```diff
@@ -49,11 +143,12 @@ enhancements that I've implemented or proposed to implement for UniCa$h.
     * Added documentation for `DeleteTransaction` [to be added]()
     * Added documentation for `ClearTransactions` [to be added]()
 * Developer Guide:
-    * Migrated details of the Developer Guide from an external source and consolidated them in GitHub.
+    * Migrated details of the Developer Guide from an external source (Google Docs) 
+  and consolidated them in GitHub.
 
     
 #### Project Management
-* Managed release v1.2 on GitHub - Can be found [here](https://github.com/AY2324S1-CS2103-T16-3/tp/releases/tag/v1.2). 
+* Managed release v1.3a on GitHub - Can be found [here](https://github.com/AY2324S1-CS2103-T16-3/tp/releases/tag/v1.2).
```

Comment:
Is this link wrong since it's supposed to be for v1.3a instead of v1.2?

Reply:
Ok I've fixed it